client/packages/system/src/Item/Components/PackVariant/PackVariantInput.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { ReactElement, useEffect, useState } from 'react';
+import {
+  Select,
+  NumericTextInput,
+  useDebounceCallback,
+  Box,
+  useTranslation,
+} from '@openmsupply-client/common';
+import { usePackVariant } from '../../context';
+
+const ENTER_PACK_SIZE = -1;
+
+// This field displays a packSize number input and unit pack drop down if unit
+// pack variants exist
+
+export const PackVariantInput = ({
```

Comment:
This is very similar to the PackVariantEntryCell, just without the table elements. I think it would be best to refactor PackVariantEntryCell to use this component underneath? But it was a little tangled, so thought cleaner just to add here, refactor separately...

Reply:
I see what you mean, with styles especially, right ?

Reply:
Yeah mostly styles - also use of column methods, though I imagine these could be passed in as props fairly easily...